statsmodels/genmod/families/family.py

```diff
@@ -1287,7 +1303,8 @@ def hyp2f1(x):
 
         return (3 / 2. * (endog ** (2 / 3.) * hyp2f1(-self.alpha * endog) -
                           mu ** (2 / 3.) * hyp2f1(-self.alpha * mu)) /
-                (mu * (1 + self.alpha * mu) * scale ** 3) ** (1 / 6.))
+                (mu * (1 + self.alpha * mu) *
+                 (scale / var_weights) ** 3) ** (1 / 6.))
```

Comment:
maybe better general pattern for this:
define a tmp variable and then adjust for var_weights
e.g.
resid = (3 / 2. * (endog ** (2 / 3.) * hyp2f1(-self.alpha * endog) -
                            mu ** (2 / 3.) * hyp2f1(-self.alpha * mu)) /
                (mu * (1 + self.alpha * mu) * scale ** 3) ** (1 / 6.))
resid *= np.sqrt(var_weights)
return resid